### PR TITLE
At update progress bar

### PR DESCRIPTION
**Pull request by atainton**

### CODE REVIEW

File: db/access.php [hunk 1]
```diff
@@ -29,35 +29,46 @@
...
 
     'block/progress:myaddinstance' => array(
         'riskbitmask' => RISK_PERSONAL,
-
         'captype' => 'read',
-        'contextlevel' => CONTEXT_BLOCK,
+        'contextlevel' => CONTEXT_SYSTEM,
         'archetypes' => array(
-            'user' => CAP_ALLOW
+            'user' => CAP_ALLOW,
```
Comment (jamieandyou) [inline db/access.php:71]: Is that comma supposed to be there?

File: db/access.php [hunk 2]
```diff
@@ -29,35 +29,46 @@
...
+
+    'block/progress:showbar' => array (
+        'captype'       => 'read',
+        'contextlevel'  => CONTEXT_BLOCK,
+        'archetypes'    => array (
+            'teacher'        => CAP_ALLOW,
+            'editingteacher' => CAP_ALLOW,
+            'student'        => CAP_ALLOW,
+            'manager'        => CAP_ALLOW,
+            'coursecreator'  => CAP_ALLOW,
```
Comment (jamieandyou) [inline db/access.php:49]: Remove comma at end

File: db/access.php [hunk 3]
```diff
@@ -29,35 +29,46 @@
...
+        'contextlevel'  => CONTEXT_BLOCK,
         'archetypes'    => array (
-            'teacher'           => CAP_ALLOW,
-            'editingteacher'    => CAP_ALLOW,
-            'manager'           => CAP_ALLOW,
-            'coursecreator'     => CAP_ALLOW
+            'teacher'        => CAP_ALLOW,
+            'editingteacher' => CAP_ALLOW,
+            'manager'        => CAP_ALLOW,
+            'coursecreator'  => CAP_ALLOW,
```
Comment (jamieandyou) [inline db/access.php:37]: Remove comma at end

File: db/access.php (hunk 1 above)
Comment (atainton) [inline db/access.php:71]: this is part of the update so i think so

**Pull request merged**